### DISCUSSION TITLE
[False-Positive] Virustotal showing Trojan?

### DISCUSSION THREAD

**Discussion by hassantaghzouti** [elmoiv/elmocut#41] [General]
virustotal.com

detectes mutiple viruses in this project 

https://www.virustotal.com/gui/file/1a27e627617f71701cab9412b1e2368d289eb38bb1596a06c1863faa55004b10

scan link
**3 security vendors and no sandboxes flagged this file as malicious**

resuls 👎 

Trojan.PSW.Python.gk
Trojan/Generic.ASMalwS.7708

**Comment by elmoiv** [comment]
A False Positive.

elmoCut is built using PyInstaller packager. Because the application is not digitally signed, this may appear due to some techniques used while building elmoCut to executable.

You can build it from source and test it yourself.
